reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/greenfield.json

```diff
@@ -309,7 +306,7 @@
       "DependsOn": ["Nat"],
       "Type": "AWS::EC2::Subnet",
       "Properties": {
-        "AvailabilityZone": {"Fn::Select": ["2", {"Ref": "SubnetAvailabilityZones"}]},
+        "AvailabilityZone": {"Fn::Select": ["2", { "Fn::GetAZs" : "" }]},
```

Comment:
Im trying to find a doc that lists regions that have both classic and vpc from there we could use conditials or when statements for the time being. I wonder if cutting over hard to boto3 for the look up script could help us out.